### PR TITLE
[fix TODO] test cluster before run e2e in cluster

### PR DESCRIPTION
**Pull request by stewart-yu**
**What this PR does / why we need it**:
fix TODO: move code from `hack/ginkgo-e2e.sh` to `cluster/test-e2e.sh`

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: cluster/test-e2e.sh [hunk 1]
```diff
@@ -14,20 +14,156 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Tests a running Kubernetes cluster.
-# TODO: move code from hack/ginkgo-e2e.sh to here
```
Comment (roberthbailey) [inline cluster/test-e2e.sh]: The TODO is to move the code, but I only see additions (and not deletions from the other file). Is there a reason we can't delete the corresponding code in ginkgo-e2e.sh? If not, we at least need to add a TODO there around the corresponding block to clean it up (ideally with information about what is preventing us from doing it now).

Comment (stewart-yu) [reply cluster/test-e2e.sh]: great,  will try

**Comment by stewart-yu** [comment]
kindly ping all to review, thanks
/assign @mikedanese 
/assign @lavalamp 
/assgin @shashidharatd 
/assgin @wojtek-t

**Pull request closed without merging**